src/bridge/cockpitpeer.c

```diff
@@ -480,8 +482,18 @@ on_transport_control (CockpitTransport *transport,
     }
   else if (self->inited)
     {
-      if (g_str_equal (command, "logout") ||
-          g_str_equal (command, "kill"))
+      if (g_str_equal (command, "logout"))
```

Comment:
These messages are swallowed until `self->inited` is true.  We might need to queue them up and replay them when the peer is inited, like we do with `open` messages.